Makefile

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env make
+# Root Makefile for IronOS - Soldering Iron Open Source Firmware Project
+
+
+### global adjustable variables
+
+# command for docker-compose tool
+ifndef DOCKER_COMPOSE
+DOCKER_COMPOSE:=docker-compose

```

Comment:
While we are here, can we test for `docker-compose` and if its not here fall back to `docker compose`

Reply:
Thoughtful suggestion. No problem, I will test & check how to make it in a more or less reliable way.